app/src/main/java/org/stepic/droid/ui/fragments/CourseSearchFragment.kt

```diff
@@ -70,6 +72,11 @@ class CourseSearchFragment: CourseListFragmentBase() {
 
     override fun onDestroyView() {
         searchCoursesPresenter.detachView(this)
+        searchView?.let {
+            searchSuggestionsPresenter.detachView(it)
```

Comment:
why presenter is not attached, when view is null?

Reply:
So, when presenter will not know about android view, this behaviour will be fixed